model/browser/events.yaml

```diff
@@ -0,0 +1,54 @@
+groups:
+  - id: event.browser.exception
```

Comment:
Similar to comments I left on #1941, I think other client instrumentation would benefit from also having these relaxed to be non-browser. Alternatives: `client.exception`, `app.exception`, `runtime.exception`.....

Reply:
@Karlie-777 can you please review https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2751 which is an initial implementation of a browser "event" which is using the existing top-level 3 attributes.